frontend/assets/javascripts/src/modules/form/stripe.es6

```diff
@@ -1,11 +1,10 @@
 import * as payment from 'src/modules/payment';
-import bean from 'bean'
 import $ from '$'
 export function init() {
     let handler = window.StripeCheckout.configure(guardian.stripeCheckout);
     let success = false;
     const button = $('.js-stripe-checkout');
-    bean.on(window, 'popstate', handler.close);
+    $(document).on('popstate', handler.close);
```

Comment:
What's the difference between listening for this event on the `window` vs the `document`?